final_task/MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+include rss_reader\font\dejavusans.ttf
+recursive-include rss_reader\templates *
```

Comment:
This will make your Iteration 4 work only for Windows when installed using setuptools, because Linux OS has different path separators.